### ISSUE TITLE
studio(a11y): update button not accessible via keyboard only on homepage

### ISSUE THREAD

**Issue by donroyco** [deckgo/deckdeckgo#1027]
### Affected part of DeckDeckGo

- [x] Studio

### Reproduction

Steps to reproduce:

1. Go to www.deckdeckgo.com when a new version has been deployed
2. Message is shown of updated version

<img width="769" alt="Screenshot 2020-12-02 at 10 39 21" src="https://user-images.githubusercontent.com/1763537/100913710-6f4c9b00-34d2-11eb-83ca-17b9bf73da0e.png">

_Notification of updated version_

3. Navigate to 'update' button with keyboard only.
 
### Expected Behavior

To navigate directly to update notification, because this is the 'best action' to do.

### Actual Behavior

Navigating by keyboard is passing by the actual notification. You can never reach it.

### Environment

- Browser(s): Chrome, latest version
- Operating System (e.g. Windows, macOS, Ubuntu): macOS

**Comment by peterpeterparker** [comment]
Cool idea 👍. It's a default Ionic Toast, it has to be checked if it is possible.
